lib/cli/cordova.js

```diff
@@ -36,6 +36,17 @@ module.exports = function(argv, callback) {
     
     // stop cordova-cli from tracking us!
     cordovaCommand.push("--no-telemetry");
+    
+    // append --save by default to phonegap plugin add 
+    var pluginCommands = ["add", "remove", "rm"];
+    var pluginAlias = ["plugin","plugins"];
+    if(cordovaCommand[0] == "cordova" && 
+       pluginAlias.indexOf(cordovaCommand[1]) != -1 && 
+       cordovaCommand.indexOf("--save") == -1 && cordovaCommand.indexOf("--no-save") == -1 &&
+       pluginCommands.indexOf(cordovaCommand[2]) != -1)
+    {
```

Comment:
This bracket should be at the end of line 46, not on a new line. Look at the formatting of other if statements in the file.